frontend/packages/helm-plugin/src/topology/helm-data-transformer.ts

```diff
@@ -225,3 +231,12 @@ export const getHelmTopologyDataModel = () => {
     return Promise.resolve(getHelmGraphModelFromMap(helmResourcesMap, resources));
   };
 };
+
+export const useHelmResources: ExtensionHook<WatchK8sResources<any>> = (namespace: string) => {
+  const resources = React.useMemo(
+    () => ({ secrets: { isList: true, kind: 'Secret', namespace, optional: true } }),
+    [namespace],
+  );
+
+  return [resources, true, undefined];
```

Comment:
return this array from useMemo itself

```suggestion
  const resources = React.useMemo(
    () => [{ secrets: { isList: true, kind: 'Secret', namespace, optional: true } }, true, undefined],
    [namespace],
  );

  return resources;
```